client/logs/bdqmnrroo3suave89msg.log

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I don't think these files supposed to be under version control. Missing a `.gitignore` directive?